scripts/zones/La_Theine_Plateau/npcs/FallenEgg.lua

```diff
@@ -0,0 +1,41 @@
+-----------------------------------
+--  Area: La Theine Plateau
+--  NPC: FallenEgg
+-----------------------------------
+package.loaded["scripts/zones/La_Theine_Plateau/TextIDs"] = nil;
+-----------------------------------
+
+require("scripts/zones/La_Theine_Plateau/TextIDs");
+
+-----------------------------------
+-- onTrigger
+-----------------------------------
+
+function onTrigger(player,npc)
+    local random = math.random(1,100);
+    local FallenEgg = 17195583;
+    local Nihniknoovi = 17195475;
+
+    local X = npc:getXPos();
+    local Y = npc:getYPos();
+    local Z = npc:getZPos();
+
+    if (random <= 20 and GetMobAction(Nihniknoovi) == ACTION_NONE) then
+        GetMobByID(Nihniknoovi):setSpawn(X,Y,Z);
+        SpawnMob(Nihniknoovi):updateClaim(player);
+        UpdateNPCSpawnPoint(FallenEgg,1800);
+    elseif (random > 20 and random <= 61) then
+        player:messageSpecial(NOTHING_OUT_OF_ORDINARY); -- missing correct TextID
```

Comment:
<field name="index">7811</field>
    <field name="text">There is a broken egg on the ground here.
Perhaps there is a nest in the boughs of this tree.≺Prompt≻

Is that what you're looking for?